package.json

```diff
@@ -23,7 +23,6 @@
     "gatsby-plugin-material-ui": "^2.1.10",
     "gatsby-plugin-react-helmet": "^3.3.13",
     "gatsby-plugin-sharp": "^2.6.39",
-    "gatsby-transformer-json": "^2.4.14",
```

Comment:
Have already updated this. Remove this change.

Reply:
Sorry, now it is resolved